kitsune/sumo/static/sumo/js/device-migration-wizard.js

```diff
@@ -6,18 +6,106 @@ class SignInStep extends BaseFormStep {
   get template() {
     return `
       <template>
-        <div>
-          <p>This is where the sign in/sign up form will go.</p>
-          <p id="email"></p>
+        <div id="sign-in-step-root">
+          <p>
+            ${gettext("Body provide short value of the account other than the bookmarks/password/history line, what else could they get out of it that would be good to know here?")}
+          </p>
+
+          <p class="for-sign-up">
+            <strong>${gettext("Create an account")}</strong>
+          </p>
+          <p class="for-sign-in">
+            <strong>${gettext("Sign in to your account")}</strong>
+          </p>
+
+          <form method="get">
+            <input name="service" value="sync" type="hidden"/>
+            <input name="action" value="email" type="hidden"/>
+            <input name="context" value="" type="hidden"/>
+            <input name="flow_id" value="" type="hidden"/>
+            <input name="flow_begin_time" value="" type="hidden"/>
+            <input name="utm_source" value="" type="hidden"/>
+            <input name="utm_medium" value="" type="hidden"/>
+            <input name="entrypoint" value="" type="hidden"/>
+            <input name="redirect_immediately" value="true" type="hidden"/>
+            <input name="redirect_to" value="" type="hidden"/>
+
+            <label for="email">${gettext("Email")}</label>
+            <input id="email" name="email" type="email" required="true" placeholder="${gettext("Enter your email")}"/>
+
+            <input class="for-sign-up" type="submit" value="${gettext("Sign up")}"/>
+            <input class="for-sign-in" type="submit" value="${gettext("Sign in")}"/>
+          </form>
+
+          <p class="for-sign-up">
+            ${gettext("Already have an account? <a class='alternative-link' href='#'>Sign in</a>")}
+          </p>
+          <p class="for-sign-in">
+            ${gettext("Don’t have an account? <a class='alternative-link' href='#'>Sign up</a>")}
+          </p>
```

Comment:
Hey @escattone - is this sort of thing, putting a tag like <a> into gettext, considered acceptable?

Reply:
@mikeconley It's legal to do that, but I think in most cases it's avoided, since it exposes things like the class name text (`alternative-link`) to localization by some who may not understand the context and know where to stop translating.

In the suggestion below, I also added the `data-event-*` attributes so we get automatic GA reporting when the link is clicked.

```suggestion
          <p class="for-sign-in">
            ${gettext("Don’t have an account?")}<a class='alternative-link' href='#' data-event-category="device-migration-wizard" data-event-action="click" data-event-label="signup-link">${gettext("Sign up")}</a>
          </p>
```